pod/perldelta.pod

```diff
@@ -21,11 +21,17 @@ XXX Any important notices here
 
 =head1 Core Enhancements
 
-XXX New core language features go here.  Summarize user-visible core language
-enhancements.  Particularly prominent performance optimisations could go
-here, but most should go in the L</Performance Enhancements> section.
+=head2 iterating over multiple values at a time.
```

Comment:
nit: Usually these headings don't end with a period

Reply:
Well spotted. Removed - it's now the same as the others in the same document.